typings.d.ts

```diff
@@ -0,0 +1,7 @@
+declare module '*.scss' {
+  interface IClassNames {
+    [className: string]: string
+  }
+  const classNames: IClassNames;
+  export = classNames;
+}
```

Comment:
Does this have to be in project root?

Reply:
Maybe not, I don't know actually. The example in the documentation had it in the root, but we could try to move it if you have a better location for it

Reply:
The location depends on which file uses this? There is also images.d.ts, it should be moved too.

Reply:
I can try to move them to a common place such as /typings/ or similar

Reply:
Okay, do that and my review is ok!